src/libs/SearchUtils.ts

```diff
@@ -592,6 +618,9 @@ function buildQueryStringFromFilterFormValues(filterValues: Partial<SearchAdvanc
     const {type, status, policyID, ...otherFilters} = filterValues;
     const filtersString: string[] = [];
 
+    filtersString.push(`${CONST.SEARCH.SYNTAX_ROOT_KEYS.SORT_BY}:${CONST.SEARCH.TABLE_COLUMNS.DATE}`);
+    filtersString.push(`${CONST.SEARCH.SYNTAX_ROOT_KEYS.SORT_ORDER}:${CONST.SEARCH.SORT_ORDER.DESC}`);
```

Comment:
> highlight on the saved search item will be lost when we change the sort order

I tend to lean toward idea that this is not a bug. For me searches differing only with sortBy or sortOrder are two different queries. Reason:
1. There might be a user which has about 1000 expenses with category 'Advertising'
2. When he queries for category 'Advertising' he will get a paginated result with only 50 of newest expenses
3. When we will change sortOrder then he will get 50 oldest expenses. These two queries are giving us different responses

Reply:
I agree that those are different queries. We should treat these params the same way we do with status, since to @rayane-djouah's point, we use the contextual filters to change these values instead of manually typing them in.